examples/airflow/README.md

```diff
@@ -1,43 +1,272 @@
+t2 = PostgresOperator(
+    task_id='inc',
+    postgres_conn_id='example_db',
+    sql='''
+    INSERT INTO counts (value)
+         VALUES (%(value)s)
+    ''',
+    parameters={
+      'value': random.randint(1, 10)
+    },
+    dag=dag
+)
 
-### Operators
+t1 >> t2
+
+```
+
+## Step 2.2: Create DAG `sum`
+
+Under `dags/`, create a file named `sum.py` and copy in the following code:
+
+```python
+from marquez_airflow import DAG
+from airflow.operators.postgres_operator import PostgresOperator
+from airflow.utils.dates import days_ago
+
+default_args = {
+    'owner': 'datascience',
+    'depends_on_past': False,
+    'start_date': days_ago(1),
+    'email_on_failure': False,
+    'email_on_retry': False,
+    'email': ['datascience@example.com']
+}
+
+dag = DAG(
+    'sum',
+    schedule_interval='*/5 * * * *',
+    catchup=False,
+    is_paused_upon_creation=False,
+    default_args=default_args,
+    description='DAG that sums the total of generated count values.'
+)
+
+t1 = PostgresOperator(
+    task_id='if_not_exists',
+    postgres_conn_id='example_db',
+    sql='''
+    CREATE TABLE IF NOT EXISTS sums (
+      value INTEGER
+    );''',
+    dag=dag
+)
+
+t2 = PostgresOperator(
+    task_id='total',
+    postgres_conn_id='example_db',
+    sql='''
+    INSERT INTO sums (value)
+        SELECT SUM(c.value) FROM counts AS c;
+    ''',
+    dag=dag
+)
+
+t1 >> t2
+
+```
+
+At this point, you should have the following under your `examples/airflow/` directory:
+
+```
+.
+├── dags
+│   ├── counter.py
+│   └── sum.py
+├── docker/
+├── docs/
+├── marquez.env
+└── requirements.txt
+```
+
+## Step 3: Start Airflow with Marquez
+
+Now that we have our DAGs defined using Marquez, let’s run the example! To start Airflow, run:
+
+```bash
+$ docker-compose up
+```
+
+> **Tip:** Use `-d` to run in detached mode.
+
+**The above command will:**
+
+* Start Airflow and install `marquez-airflow` used to collect DAG metadata
+* Start Marquez
+* Start Postgres
+
+To view the Airflow UI and verify it's running, open http://localhost:8080. Then, login using the username and password: `airflow` / `airflow`. You can also browse to http://localhost:3000 to view the Marquez UI.
+
+## Step 4: View Collected Metadata
+
+To view DAG metadata collected by Marquez from Airflow, browse to the Marquez UI by visiting http://localhost:3000. Then, use the _search_ bar in the upper right-side of the page and search for the `counter.inc` job. To view lineage metadata for `counter.inc`, click on the job from the drop-down list:
+
+> **Note:** If the `counter.inc` job is not in the drop-down list, check to see if Airflow has successfully executed the DAG.
+
+![](./docs/search.png)
+
+If you take a quick look at the lineage graph for `counter.inc`, you should see `public.counts` as an output dataset and `sum.total` as a downstream job!
+
+![](./docs/lineage-view-job.png)
+
+## Step 5: Troubleshoot a Failing DAG with Marquez
```

Comment:
I think this is a good user story and we should highlight it in its own piece rather than have it here

Reply:
Good idea, I've opened an issue to help breakup the examples : https://github.com/MarquezProject/marquez/issues/1108